src/components/Game/Timer/Timer.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import classes from './Timer.scss';
+import { computeMinutes, computeSeconds, printTwoDigits } from '../../../helpers';
+
+class Timer extends Component {
+    state = {
+        // 60 sec. only for testing reasons. it's gonna be more :)
+        seconds: 60,
+    };
+
+    componentDidMount() {
+        this.startTimer();
+    }
+
+    componentWillUnmount() {
+        this.stopTimer();
+    }
+
+    startTimer = () => {
+        const { timeIsOver } = this.props;
+
+        this.interval = setInterval(() => {
+            const { seconds } = this.state;
+
+            this.decrement();
+            if (seconds <= 1) {
+                this.stopTimer();
+                timeIsOver();
+            }
+        }, 1000);
```

Comment:
thought about it but what if there's some delay between getting `Date` and starting the game?

Reply:
> what if there's some delay between getting Date and starting the game?

:thinking:  well, surely the way component is loaded/mounted/rendered leads to some imprecision of time measurement but it can't be helped and I believe it's not a problem we should worry about

the solution relying on `setInterval + decrementing this.state.seconds by interval` will work properly most of the time but the more I think about it the more I consider it a suboptimal (:P) idea because it can lead to situations where time measurement is noticeably inaccurate (e.g. the device freezes a bit and affects the delay of `setInterval`)

`setInterval + decrementing this.state.seconds by interval` solution is like gathering sixty people, making them to form a line and asking them to poke the neighbor exactly one second after you are poked, it will work but it's fragile
`setInterval + computing time elapsed from t0` is more like a single person checking their watch every second, more acurate/stable/trustworthy

Reply:
sixty people poking one another 💛😂
okay, I computed time elapsed. hope it's correct.